SwrveSDK/src/main/java/com/swrve/sdk/SwrveLogger.java

```diff
@@ -0,0 +1,165 @@
+package com.swrve.sdk;
+
+import android.util.Log;
+
+import com.swrve.sdk.SwrveLogger;
+
+public class SwrveLogger {
+
+    private static final String LOG_TAG = "SWRVE";
+
+    // Config/Active/Inactive
+    private static boolean isActive = true;
```

Comment:
Would BuildConfig.debug be a better default?
private static boolean isActive = BuildConfig.debug;